Funny enough run into this issue in my wf-core PR

```
[ERROR] Failed to execute goal org.wildfly.glow:wildfly-glow-arquillian-plugin:2.1.0.Final:scan (scan-legacy-sar-layers) on project wildfly-ts-integ-basic: Error in glow discovery.
[ERROR] -Expected: [ee-integration, mail, naming, sar, transactions]==>ee-core-profile-server,mail,sar,transactions
[ERROR] -Found   : [ee-integration, mail, naming, resource-adapters, sar, transactions]==>ee-core-profile-server,mail,resource-adapters,sar
[ERROR] -> [Help 1]
```

This seems to be hitting a bug in glow. The issue is that the layer discovery is non-deterministc. Maybe because it iterates over collections that are not deterministically ordered and then breaks... so bit of a reminder to myself or anyone to file and fix this.

--- a/pom.xml
+++ b/pom.xml
@@ -519,7 +519,7 @@
         <legacy.version.org.hibernate>6.6.53.Final</legacy.version.org.hibernate>
         <!-- We need to use this value in configuring maven plugins, so simply overriding version.org.hibernate
              in boms/standard-preview-ee is inadequate. So we'll declare the value here and reference it in boms/standard-preview-ee. -->
-        <standard.version.org.hibernate>7.4.1.Final</standard.version.org.hibernate>
+        <standard.version.org.hibernate>7.4.3.Final</standard.version.org.hibernate>
         <version.org.hibernate.models>1.1.1</version.org.hibernate.models>
         <version.org.hornetq>2.4.11.Final</version.org.hornetq>
         <version.org.infinispan>16.0.13</version.org.infinispan>

